Parse/ParseClientConfiguration.h

```diff
@@ -10,6 +10,7 @@
 #import <Foundation/Foundation.h>
 
 #import <Parse/PFConstants.h>
+#import "PFFileUploadController.h"
```

Comment:
It's because the header is 'exported' that avoids the confusion between import <module/header.h> vs import "header.h"